### ISSUE TITLE
UI text is invisible to screen readers

### ISSUE THREAD

**Issue by davecparker** [davecparker/code12#345]
UI text displayed with Corona's display.newText is not visible at all to screen readers (accessibility), including getFile screen, options screen, error messages, console, and varWatch. The web view for help is accessible however.
